src/mixins/common-mixin.ts

```diff
@@ -92,6 +92,13 @@ export class CommonMixin extends Vue {
     }
   }
 
+  /** Returns true if the specified alteration NR is allowed to be done online. */
+  isAlterOnline (type: NrRequestTypeCodes): boolean {
+    return !(type === NrRequestTypeCodes.CONVERT_BEN ||
+      type === NrRequestTypeCodes.CONVERT_CORP ||
+      type === NrRequestTypeCodes.CONVERT_ULBE)
+  }
```

Comment:
Do you think we should use a FF for this, so that we can expand it easily in the future?

Reply:
I don't think we should use FF in this case. Because now anything that is from/to BEN cannot be done online, once it can be done, we can remove this part.

Reply:
But that's the point of a FF: we can enable it in real-time without needing to change any code and deploy.
Maybe check with Mihai.

Reply:
Summary of what we discussed:

The FF will say which request type codes (eg, BECV, UC) will show "Alter Now" instead of "Go To COLIN"... **but** you also have code that says "from or to BEN will show a special Contact Us panel".

So, BECV will show the panel, UC will show Alter Now, but ULCB (not in FF atm) will show Go To COLIN.

Reply:
Hongjing, please verify that your code matches what we discussed, and then you can resolve this conversation. Thanks.

Reply:
Yes, verified!